File: catalog/admin/cat/delete.php
<?php
defined('AUTH') or die('Restricted access');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogCat.php';

$id = intval($_POST['id']);
if($id == 0 || $_POST['agree'] != 'yes')
	header('location: /admin/com/catalog');

$catalog = new AdminCatalogCat();
$catalog->delete($id);

header('location: /admin/com/catalog');
exit;

File: dan/system/catalog/cat/delete.py
import sys
sys.path.append('system/catalog/classes')
from system.catalog.classes.Catalog import Catalog


def delete(SITE):
    print('FUNCTION -> system-> calalog -> cat -> delete')

    if 'agree' in SITE.post and SITE.post['agree'] == 'yes':
        CATALOG = Catalog(SITE)
        catalog = CATALOG.delete(SITE.post['id'])  # Удаляем текущий элемент

    return {'redirect': '/system/catalog/cat'}